UPGRADE.md

```diff
@@ -1209,6 +1209,35 @@ Besides, some props supported by the previous implementation aren't anymore:
 
 `useGetMainList` was a modified version of `useGetList` designed to keep previous data on screen upon navigation. As [this is now supported natively by react-query](https://react-query.tanstack.com/guides/paginated-queries#better-paginated-queries-with-keeppreviousdata), this hook is no longer necessary and has been removed. Use `useGetList()` instead.
 
+## The MUI `<ThemeProvider>` is not set by `<Layout>` anymore
+
+This shouldn't impact your code unless you had a completely custom `<Layout>` component. If you do but still uses the default `<Admin>` component, you can safely remove the `ThemeProvider` from your `Layout`:
```

Comment:
This is a bit misleading, so developers may wonder if they need to manually add the ThemeProvider themselves. You should explain that ThemeProvider is set by AdminContext rather than Layout

Reply:
Not sure why github shows the old version but I added a sentence before this one as requested